src/Main.hs

```diff
@@ -1,12 +1,13 @@
-import qualified Data.Text as T (strip, pack, unpack)
-
 main = putStrLn "It works!"
 
-wrap :: String -> [String]
-wrap = map trim . combine . splitAt 80
+wrap :: Int -> String -> [String]
+wrap n s = foldl (splitWordsAt n) [] . words $ s
```

Comment:
This _was_ point free and I was super happy about it, but then that pesky `n` showed up and screwed everything up. Good call re: `$` though. Makes much more sense that way.